masonry/src/render_root.rs

```diff
@@ -559,6 +557,31 @@ impl RenderRoot {
         run_mutate_pass(self, widget_state);
     }
 
+    pub(crate) fn request_render_all(&mut self) {
+        fn request_render_all_in(
+            mut widget: ArenaMut<'_, Box<dyn Widget>>,
+            state: ArenaMut<'_, WidgetState>,
+        ) {
+            state.item.needs_paint = true;
+            state.item.needs_accessibility = true;
+            state.item.request_paint = true;
```

Comment:
With the current state of Masonry, I'm less sure that we need a repaint on a rescale. The logical size of everything is still the same, and there is no way for a widget to access the UI scale in its painting method.

(Accessibility does need it, as the accesskit rectangles work on physical coordinates)

Reply:
I'm still hazy on how scale even works in Masonry. One of my todo items before next release is to go all scale-adjacent code and document out semantics.